code/modules/hallucinations/effects/major.dm

```diff
@@ -408,3 +408,99 @@
 	if(images && images[1])
 		images[1].icon = 'icons/mob/alien.dmi'
 		images[1].icon_state = "alienh_pounce"
+
+/**
+  * # Hallucination - Sniper
+  *
+  * Fires a penetrator round at the target. On hit, knockdown + stam loss + hallucinated blood splatter for a bit.
+  */
+/obj/effect/hallucination/sniper
+	duration = 15 SECONDS
+
+/obj/effect/hallucination/sniper/Initialize(mapload, mob/living/carbon/target)
+	. = ..()
+
+	// Find a start spot for the sniper bullet
+	var/list/possible_turfs = list()
+	for(var/turf/t in RANGE_TURFS(13, target.loc))
+		possible_turfs += t
```

Comment:
If you want to use a single-letter variable name, please make sure it's uppercase!
```suggestion
	for(var/turf/T in RANGE_TURFS(13, target.loc))
		possible_turfs +=T
```